python_modules/dagster/dagster/_core/execution/context/system.py

```diff
@@ -206,6 +206,10 @@ def step(self) -> ExecutionStep:
     def node_handle(self) -> "NodeHandle":
         raise NotImplementedError()
 
+    @property
+    def op_retry_policy(self) -> Optional[RetryPolicy]:
+        return self.job.get_definition().get_retry_policy_for_handle(self.node_handle)
```

Comment:
should we leave a note here - "should be able to move retry policy to snapshots if trying to do host process execution"